packages/contracts/.env.example

```
@@ -50,3 +50,24 @@ OWNER_PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f
 # - testnet: https://sepolia.gateway.tenderly.co
 # - mainnet: https://mainnet.gateway.tenderly.co 
 RPC_URL="http://127.0.0.1:8545"
+
+# --- Security monitor (issue #927) ---
+# Diamond proxy address exposing UbiquityPoolFacet and AccessControlFacet functionality.
+UBQ_DIAMOND_ADDRESS=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)